cinema/views.py

```diff
@@ -101,8 +105,27 @@ def get_serializer_class(self):
         if self.action == "retrieve":
             return MovieDetailSerializer
 
+        if self.action == "create":
+            return MovieCreateSerializer
```

Comment:
The `get_serializer_class` method in `MovieViewSet` correctly returns `MovieCreateSerializer` for the 'create' action. Ensure that this serializer is properly defined to handle the creation of movies, including any necessary validation.